src/manage.py

```diff
@@ -68,21 +81,42 @@ def seed():
             print("\n\n\nFailed:\n", e, "\n\n")
 
 
-def shell():
-    """Make a shell/REPL context available."""
-    return dict(app=app,
-                db=db,
-                User=User,
-                Society=Society,
-                Activity=Activity)
+@manager.command
+def update_cohorts(name=None):
+    """Sync cohorts in DB with cohorts from Andela API."""
+    pass
 
 
-manager.add_command("shell", Shell(make_context=shell))
-manager.add_command("db", MigrateCommand)
+@manager.command
+def link_society_cohort(cohort_name, society_name):
+    """CLI tool, link cohort with society."""
+    with app.app_context():
+        cohort = Cohort.query.filter_by(name=cohort_name).first()
+        if not cohort:
+            return print(
+                f'Error cohort by name: {cohort_name} does not exit in DB.')
```

Comment:
'Error cohort by name: {cohort_name} does not exit in DB.' should be 'Error cohort by name: {cohort_name} does not exist in DB.'